assets/_sass/base/_typography.scss

```diff
@@ -48,7 +54,14 @@ h6 {
 }
 
 p {
-  margin: 0 0 ($base-line-height * .5);
+  font-size: 1em;
+  padding: 0; 
+  margin: 0;
+  line-height: 1.5;
+    + p {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented 2 spaces, but was indented 4 spaces